packages/components/color-picker/src/utils/draggable.ts

```diff
@@ -44,5 +44,5 @@ export function draggable(element: HTMLElement, options: DraggableOptions) {
   }
 
   element.addEventListener('mousedown', downFn)
-  element.addEventListener('touchstart', downFn)
+  element.addEventListener('touchstart', downFn, { passive: true })
```

Comment:
`preventDefault` is called inside the downFn function, and `passive` cannot be set to true here.